mnelab/dialogs/__init__.py

```diff
@@ -2,27 +2,31 @@
 #
 # License: BSD (3-clause)
 
-from .annotations import AnnotationsDialog
-from .append import AppendDialog
+from .append_data import AppendDataDialog
 from .calc import CalcDialog
-from .channel_properties import ChannelPropertiesDialog
-from .crop import CropDialog
+from .change_reference import ChangeReferenceDialog
+from .create_epochs import CreateEpochsDialog
+from .crop_data import CropDataDialog
 from .drop_bad_epochs import DropBadEpochsDialog
-from .epoch import EpochDialog
-from .erds import ERDSDialog, ERDSTopomapsDialog
+from .edit_annotations import EditAnnotationsDialog
+from .edit_channel_properties import EditChannelPropertiesDialog
+from .edit_events import EditEventsDialog
```

Comment:
Same here, I'd prefer `EventsDialog` (and `events.py`). I think the reason for keeping these names is that these dialogs are also used for viewing information, not just editing. I know they are in the Edit menu, but we don't have to religiously follow a naming scheme when we have reasons to deviate.